.gitignore

```diff
@@ -6,7 +6,6 @@ __pycache__/
 # Environments
 .env
 *.venv*
-*.env*
```

Comment:
In backend folder we have .env.example file which has all the env variables, for local envs we have .env.local. I've removed .env file here because it was unproperly handled by git, so .env.example was also ignored